lib/game_teams_collection.rb

```diff
@@ -0,0 +1,15 @@
+require 'csv'
+
+class GameTeamsCollection
+  attr_reader :game_teams
+  def initialize(game_teams_csv)
+    @game_teams = create_game_teams(game_teams_csv)
+  end
+
+  def create_game_teams(game_teams_csv)
+    game_teams_data = CSV.read(game_teams_csv, headers: true, header_converters: :symbol)
```

Comment:
I agree on this being a module, could save us lots of repetitive code. We'll handle it when we get there